noether/src/tool_path_planner_server.cpp

```diff
@@ -159,7 +158,7 @@ class TppServer
             }
             tp.segments.push_back(seg);
           }
-          trp.paths.push_back(tp);
+	  result.tool_paths[i].paths.push_back(tp);
```

Comment:
Really not sure what `trp` was doing here, which makes me suspicious of putting the result somewhere else... But something was clearly smelly with the existing code.